### PR TITLE
Update it_IT.po

### PR DESCRIPTION
**Pull request by StarFang208**
New translated lines and translator's name fixed.

### CODE REVIEW

File: CliClient/locales/it_IT.po [hunk 1]
```diff
@@ -3305,212 +2970,3 @@ msgstr "Accedi a OneDrive"
...
-#~ msgstr "Titolo del \"Cose-da-fare\":"
-
-#~ msgid "\"%s\": \"%s\""
-#~ msgstr "\"%s\": \"%s\""
-
-#~ msgid "Delete notebook?"
-#~ msgstr "Eliminare il taccuino?"
-
-#~ msgid "File system synchronisation target directory"
-#~ msgstr "Cartella di destinazione per la sincronizzazione nel file system"
```
Comment (tessus) [inline CliClient/locales/it_IT.po]: As mentioned before: Do not remove these commented translations.

**Comment by tessus** [comment]
This is definitely a lot better, but it's not based on git master but from a git hash a few weeks old.
Ad you can see in the diff, the locations in the file's comments do not exist, which means you branched it off at a point where the `app` directory still existed in `ElectronClient`.

I think I can merge it, after you have restored the deleted commented out translations at the end of the file.

But please, for the future: create a branch from git master (pull upstream first). I will never ever merge something like this again.

After this is merged, delete all your local and remote branches which you used for translating.

File: CliClient/locales/it_IT.po (hunk 1 above)
Comment (tessus) [reply CliClient/locales/it_IT.po]: Hello. Are you still there?

Comment (StarFang208) [reply CliClient/locales/it_IT.po]: Sorry, I'm going to delete the branches now.

Just give me a motent to find them.

Comment (tessus) [reply CliClient/locales/it_IT.po]: No worries For this PR, you only have to add the deleted lines again. That's all.

For the future you will have to create new branches and start from a point that is HEAD at git master.

**Comment by tessus** [comment]
Thanks for the update.

Please keep in mind to always use a new branch (with the latest git master) for new PRs!

**Pull request merged**